anvil-recyclerview-v7/src/main/kotlin/dev/inkremental/dsl/androidx/recyclerview/RenderableRecyclerViewAdapter.kt

```diff
@@ -0,0 +1,67 @@
+package dev.inkremental.dsl.androidx.recyclerview
+
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.recyclerview.widget.RecyclerView
+import dev.inkremental.Anvil.Mount
+import dev.inkremental.Anvil.render
+import dev.inkremental.Renderable
+import dev.inkremental.RenderableAdapter
+
+
+abstract class RenderableRecyclerViewAdapter : RecyclerView.Adapter<RenderableRecyclerViewAdapter.MountHolder>() {
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MountHolder {
+        val root = FrameLayout(parent.context)
+        root.layoutParams = FrameLayout.LayoutParams(
+                ViewGroup.LayoutParams.MATCH_PARENT,
+                ViewGroup.LayoutParams.WRAP_CONTENT)
```

Comment:
Let's use our `MATCH` and `WRAP` here :)

Reply:
Hmm, but WRAP and MATCH now  are subclasses of Size sealed class. Or do you mean create a new constants?

Reply:
Nah, I forgot they've changed. Nevermind.